src/main/java/bio/terra/workspace/app/configuration/external/CrlConfiguration.java

```diff
@@ -0,0 +1,115 @@
+package bio.terra.workspace.app.configuration.external;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.common.cleanup.CleanupConfig;
+import bio.terra.cloudres.google.billing.CloudBillingClientCow;
+import bio.terra.cloudres.google.cloudresourcemanager.CloudResourceManagerCow;
+import bio.terra.cloudres.google.serviceusage.ServiceUsageCow;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.time.Duration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+/** Configuration to use Terra Cloud Resource Library. */
+@Component
+@EnableConfigurationProperties
+@EnableTransactionManagement
+@ConfigurationProperties(prefix = "workspace.crl")
```

Comment:
Are these settings supposed to be used outside of the test configuration, or is that a future PR?

Reply:
The beans produced by the configuration are supposed to be used outside of tests in a future PR. So far, there's no configuration that needs to be set outside of testing.
But yes, this isn't used yet.